model_mommy/mommy.py

```diff
@@ -417,7 +454,7 @@ def get_required_values(generator, field):
     If required value is a string, simply fetch the value from the field
     and return.
     '''
-    #FIXME: avoid abreviations
+    # ¬FIXME: avoid abreviations
```

Comment:
I might be missing something but why do we need this symbol? 😅

Reply:
@anapaulagomes I wonder what is up with my personal laptop lately with all that symbols appearing here and there, really 😳